Foundation/HTTPCookie.swift

```diff
@@ -377,6 +377,15 @@ open class HTTPCookie : NSObject {
         }
     }
 
+    private class func  getStringValue(strVal: Any?) -> String? {
```

Comment:
Three suggestions here:
1. Though this function name serves the purpose, we could shorten it, may be something like `stringValue`. Just a suggestion :)
2. Could this function be nested inside the initialiser where it is being used?
2. Because we only seem to be adding support for Substrings, do we really need the `strVal as? String` check?